next-unbuilt-package.sh

```diff
@@ -61,14 +61,11 @@ for CRATE in $POPULAR_CRATES; do
         echo "${CRATE}-${VERSION}-${TARGET_ARCH}.tar.gz already uploaded. Keep going." 1>&2
```

Comment:
Higher up in this file: where does START_AFTER_CRATE come from after this change? (it currently comes from the last commit on trigger/*)

Reply:
I removed `START_AFTER_CRATE` because I'm not sure what is the use of it.
It seems to me it is easier just removing it.

Reply:
I think with #174 , we should be good without `START_AFTER_CRATE`.

Reply:
I will try to describe the problems that START_AFTER_CRATE solves shortly. I will be making diagrams on https://app.excalidraw.com/l/4NJeenUSxBH/48aT0OfmkFU if you want to follow along. I will paste them in here when I'm done.

Reply:
for posterity:

![START_AFTER_CRATE-problem-1 excalidraw](https://user-images.githubusercontent.com/254647/223373907-25dca6c8-bd84-49a4-b265-be408187c55c.png)
![START_AFTER_CRATE-problem-2 excalidraw](https://user-images.githubusercontent.com/254647/223373922-55d8e807-2aae-4368-af65-68c8f36debaa.png)
![START_AFTER_CRATE-solution-1 excalidraw](https://user-images.githubusercontent.com/254647/223373929-c5e01188-0544-41fd-83f0-3cd44b84faab.png)